html-syntax/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Syntax</title>
+  </head>
+  <body>
+    <img src="https://archives.bulbagarden.net/media/upload/a/ad/681Aegislash.png">
```

Comment:
Good job, Eric!
One small request: could you add `alt` attribute to `img` tag please?